### PR TITLE
Lightbulb effects

### PR DESCRIPTION
**Pull request by yoyobatty**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: YoYoBatty
add: Light bulbs randomly flicker when hit.
tweak: Light flicker proc is rewritten to flicker more like a damaged bulb (Changes to a dark brown colour and plays sparking + flickering noises).
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

https://i.imgur.com/n2CezvR.mp4

### CODE REVIEW

**Comment by Carbonhell** [comment]
sure why not +1

File: hippiestation/code/modules/power/lighting.dm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		visible_message("<span class='warning'>[src] begins flickering!</span>","<span class='italics'>You hear an electrical sparking.</span>")
+		for(var/i = 0; i < amount; i++)
+			if(status != LIGHT_OK)
+				break
+			on = !on
+			if(prob(18) && !on)//only spark when off so it doesn't occur too much
+				do_sparks(1, FALSE, src)
+			else if(prob(40))
+				bulb_colour = LIGHT_COLOR_BROWN
+				playsound(src, pick('sound/effects/sparks1.ogg', 'sound/effects/sparks2.ogg', 'sound/effects/sparks3.ogg', 'sound/effects/sparks4.ogg', 'sound/effects/light_flicker.ogg'), 100, 1)
```
Comment (Carbonhell) [inline hippiestation/code/modules/power/lighting.dm:22]: doesn't do_sparks already do this?

Comment (yoyobatty) [reply hippiestation/code/modules/power/lighting.dm:22]: I wanted random spark noises to play when there wasn't actual sparks being made

**Comment by JohnGinnane** [comment]
this is pretty neat

**Comment by ghost** [comment]
cool plus my one of thirds

**Comment by yoyobatty** [comment]
Will get a video up soon

**Comment by dreamcrab** [comment]
nice yea +6

**Review by Carbonhell — APPROVED** [review]

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**